src/app/fsat/fsat-tabs/fsat-tabs.component.ts

```diff
@@ -257,4 +273,67 @@ export class FsatTabsComponent implements OnInit {
       badge.display = false;
     }
   }
+
+  getCanContinue() {
+    if (this.stepTab === 'system-basics' || this.stepTab === 'fan-operations') {
+      return true;
+    } else if (this.stepTab === 'fsat-fluid') {
+      let isValid: boolean = this.fsatFluidService.isFanFluidValid(this.fsat.baseGasDensity, this.settings);
+      if (isValid) {
+        return true;
+      } else {
+        return false;
+      }
+    } else if (this.stepTab === 'fan-setup') {
+      let isValid: boolean = this.fanSetupService.isFanSetupValid(this.fsat.fanSetup, false);
+      if (isValid) {
+        return true;
+      } else {
+        return false;
+      }
+    } else if (this.stepTab === 'fan-motor') {
+      let isValid: boolean = this.fanMotorService.isFanMotorValid(this.fsat.fanMotor);
+      if (isValid) {
+        return true;
+      } else {
+        return false;
+      }
+    } else if (this.stepTab === 'fan-field-data') {
+      let isValid: boolean = this.fanFieldDataService.isFanFieldDataValid(this.fsat.fieldData);
+      if (isValid) {
+        return true;
+      } else {
+        return false;
+      }
+    }
+  }
+
+  continue() {
+    let tmpStepTab: string = this.fsatService.stepTab.getValue();
+    if (tmpStepTab === 'fan-field-data') {
+      this.fsatService.mainTab.next('assessment');
+    } else {
+      let assessmentTabIndex: number = _.findIndex(this.stepTabs, function (tab) { return tab == tmpStepTab });
```

Comment:
We can use the native array.findIndex on this small array and remove the import for lodash.